README.md

```diff
@@ -82,6 +82,7 @@ We would love it if you could help contributing to this repository.
 * [Tolbxela](https://github.com/tolbxela)
 * [Tommy Parnell](https://github.com/tparnell8)
 * [twsl](https://github.com/twsI)
+* [DongHyeon Kim](https://github.com/wplong11) ([@wplong11](https://github.com/wplong11))
```

Comment:
Just use _DongHyeon Kim_ or _wplong11_, not both.

Reply:
Modified.